specs/order_spec.rb

```diff
@@ -111,15 +111,59 @@
       expect(order.total).must_equal before_total
     end
   end
+
+  describe "#remove_product" do
+    it "Decreases the number of products" do
+      products = { "banana" => 1.99, "cracker" => 3.00 }
+      before_count = products.count
```

Comment:
Good work on these tests! I feel like you've captured all the interesting behavior of this method.